acl/policy.go

```diff
@@ -55,6 +55,25 @@ var (
 	validNamespace = regexp.MustCompile("^[a-zA-Z0-9-*]{1,128}$")
 )
 
+const (
+	// The following are the fine-grained capabilities that can be granted for
+	// node volume management.
+	//
+	// The Policy block is a short hand for granting several of these. When
```

Comment:
```suggestion
	// The Policy field is a short hand for granting several of these. When
```

Reply:
Fixed here and in the other places as well.